tests/spectron/close.spectron.js

```diff
@@ -6,7 +6,7 @@ let app = new Application({});
 describe('Tests for Close', () => {
 
     let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
```

Comment:
what `Application.getTimeOut()` does exactly? and, where does it get the timeout information from?